README.md

```diff
@@ -10,6 +10,7 @@ Live preview [here]( https://vudej.github.io/Leaderboard/dist/).
 
```

Comment:
- [ ] For a more professional README, kindly fix your screenshot issue

  ![vudej-read](https://user-images.githubusercontent.com/36542986/151152404-a21cfe0d-fb97-4679-b70f-ac7c07a770b9.jpg)